configure.py

```diff
@@ -75,6 +75,19 @@ def main(args):
             if options.mock:
                 sources += ["linux/mockfile.c"]
         build.static_library("cpuinfo", map(build.cc, sources))
+        if build.target.is_ppc64:
+            if build.target.is_linux:
+                sources += [
+                    "powerpc/cache.c",
+                    "powerpc/uarch.c",
```

Comment:
Please follow generic pattern
```suggestion
        if build.target.is_ppc64:
            source += [ "powerpc/cache.c", "powerpc/uarch.c", ]
            if build.target.is_linux:
                sources += [
```

Reply:
I have followed suggested change and followed the generic pattern